src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -95,6 +95,14 @@ public static class MachineOptions {
                 "Add the unlocalized Recipe Map name to blacklist the machine.",
                 "Default: All machines allowed"})
         public String[] processingArrayBlacklist = new String[0];
+
+        @Config.Comment({"Whether to enable the cleanroom, required for various recipes.", "Default: true"})
+        public boolean enableCleanroom = true;
+
+        @Config.Comment({"Whether multiblocks are able to run cleanroom recipes outside the cleanroom.",
```

Comment:
I think it is better to take Automatic Filteration Maintenance Hatch into account.
cf: `Whether multiblocks are able to run cleanroom recipes outside the cleanroom without any requirement.`